specification/redis/resource-manager/Microsoft.Cache/stable/2020-06-01/redis.json

```diff
@@ -2071,6 +2071,18 @@
           "$ref": "#/definitions/RedisCreateProperties",
           "description": "Redis cache properties."
         },
+        "redisVersion": {
```

Comment:
This needs to go inside RedisCreateProperties, its a property of your resource, not a common ARM property

Reply:
Moved under RedisCreateProperties